lib/browser/rpc-server.js

```diff
@@ -465,6 +465,10 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CONFIRM', function (event, message, title) {
 
 // Implements window.close()
 ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
-  event.sender.getOwnerBrowserWindow().close()
```

Comment:
I tried to follow how other methods in this file had try catch. I started with null check and changed into this based on other method implementation. Let me know if you feel we should change into this.